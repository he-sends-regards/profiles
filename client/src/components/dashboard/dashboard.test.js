import React from 'react';
import renderer from 'react-test-renderer';
import Dashboard from './dashboard.jsx';

it('Renders correctly without isActive prop', () => {
  const tree = renderer
      .create(<Dashboard isActive={false} />)
      .toJSON();
  expect(tree).toMatchSnapshot();
});

it('Renders correctly with isActive prop', () => {
  const tree = renderer
      .create(<Dashboard isActive={true} />)
      .toJSON();
  expect(tree).toMatchSnapshot();
});
