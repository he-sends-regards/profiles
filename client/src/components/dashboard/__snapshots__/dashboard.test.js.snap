// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly with isActive prop 1`] = `null`;

exports[`Renders correctly without isActive prop 1`] = `null`;
